package-lock.json

```diff
@@ -1,12 +1,13049 @@
 {
   "name": "nimbella-cli",
   "version": "1.16.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@saxena52 back out this file from the PR - it shouldn't be needed for the new tests.